src/main/java/nl/novi/techiteasycontroller/services/TelevisionService.java

```diff
@@ -18,17 +21,133 @@ public TelevisionService(TelevisionRepository televisionRepository) {
         this.televisionRepository = televisionRepository;
     }
 
-    public List<Television> getTelevisions() {
-        return televisionRepository.findAll();
+    public List<OutputTelevisionDto> getTelevisions() {
+        List<Television> televisions = televisionRepository.findAll();
+        List<OutputTelevisionDto> televisionDtos = new ArrayList<>();
+
+        for (Television television : televisions) {
+           OutputTelevisionDto dto = toDto(television);
+            televisionDtos.add(dto);
+        }
+
+        return televisionDtos;
     }
 
-    public Television getTelevision(Long id) {
+    public OutputTelevisionDto getTelevision(Long id) {
         Optional<Television> optionalTelevision = televisionRepository.findById(id);
         if (optionalTelevision.isEmpty()) {
             throw new RecordNotFoundException("Television with id " + id + " not found");
         } else {
-            return optionalTelevision.get();
+            return toDto(optionalTelevision.get());
+        }
+    }
+
+    public void saveTelevision(InputTelevisionDto dto) {
```

Comment:
Het is in de POST mapping wel handig om een id te returnen, maar dat gaan we in de les van vandaag nog kort behandelen